pkg/volume/util/fsquota/quota_linux.go

```diff
@@ -409,9 +412,11 @@ func ClearQuota(m mount.Interface, path string) error {
 	if !ok {
 		return fmt.Errorf("ClearQuota: No quota available for %s", path)
 	}
-	var err error
 	projid, err := getQuotaOnDir(m, path)
 	if projid != dirQuotaMap[path] {
+		if err != nil {
```

Comment:
Should we check `if err != nil` and then check if `projid != dirQuotaMap...`?

Reply:
@mattjmcnaughton good point. Was just assuming that in case of err != nil `projid != dirQuotaMap[path]` condition will be true, and didn't want to alter the existing behavior. I'll do error check beforehand.